suite/auto-sync/CppTranslator/Patches/NamespaceArch.py

```diff
@@ -39,4 +39,12 @@ def get_patch(self, captures: [(Node, str)], src: bytes, **kwargs) -> bytes:
                     res += namespace_struct(src, namespace_id, d) + b";\n\n"
                 case _:
                     res += get_text(src, d.start_byte, d.end_byte) + b"\n"
-        return res
+        return (
```

Comment:
Will this change also affect previously generated files for Tricore, ARMv7, PPC?

Reply:
Not semantically. It's only a comment to spot the previous namespace beginning and end. It is meant as a hint that all identifiers within this "scope" should have the namespace id prepended.

In general, all identifiers which were part of a namespace get the namespace id prepended because it eases generation. Since those identifiers are always accessed via the qualified identifier within C++ code we can translate them always like this: `AArch64PacKey::someID` -> `AArch64PacKey_someID`.
This is easier then removing the namespace id at a thousand places.

But yes. I'll update the translated files before the `v6` release so this stuff is in there as well.